src/lib/edge-functions/registry.js

```diff
@@ -178,6 +184,24 @@ class EdgeFunctionsRegistry {
     return declarations
   }
 
+  static getEnvironmentVariables(envConfig) {
+    const env = Object.create(null)
+    Object.entries(envConfig).forEach(([key, variable]) => {
+      if (
+        variable.sources.includes('ui') ||
+        variable.sources.includes('account') ||
+        variable.sources.includes('addons')
+      ) {
+        env[key] = variable.value
+      }
+    })
+
+    env.DENO_REGION = 'local'
+    env.DENO_DEPLOYMENT_ID = 'xxx='
```

Comment:
I just remembered that this is used in the bootstrap layer to detect whether we're running in production or not (see https://github.com/netlify/edge-functions-bootstrap/blob/main/src/bootstrap/environment.ts#L2).

We might want to remove this variable until we figure out a better way to do this detection.

Reply:
Good catch. I was wondering something similar too and if I should add `NETLIFY_DEV` but I figured that `DENO_REGION === 'local'` would do the same, could we use it in the bootstrap?

Anyway commenting for now.